### ISSUE TITLE
TypeError: Cannot read property 'find' of undefined

### ISSUE THREAD

**Issue by Mensix** [vuejs/vitepress#110]
**Describe the bug**
npx vitepress build command throws TypeError: Cannot read property 'find' of undefined

**To Reproduce**
* git clone https://github.com/Mensix/vitepress-error-demo.git
* cd vitepress-error-demo
* npm install
* npx vitepress build

**Expected behavior**
Pages are rendering normally.

**System Info**
- vitepress version: 0.7.0
- vite version: 1.0.0-rc.8
- Node version: 12.19.0
- OS version: Windows 10 Home, version 1903, compilation 18362.1139

**Comment by kiaking** [comment]
Confirmed. Happens same for me with the repo. I have no idea why... 😳 We should look into it.

**Comment by fjc0k** [comment]
+1

**Comment by fjc0k** [comment]
solved by locking `vite` to `1.0.0-rc.3`:

```json
{
    "pnpm": {
        "overrides": {
            "vite": "1.0.0-rc.3"
        }
    }
}
```

**Comment by Mensix** [comment]
@fjc0k can you provide guide on how to use this way?

**Comment by fjc0k** [comment]
@Mensix 
yarn: https://classic.yarnpkg.com/en/docs/selective-version-resolutions/
pnpm: https://pnpm.js.org/en/package_json#pnpmoverrides

**Comment by Mensix** [comment]
Seems like after upgrading vitepress to v0.7.1 the error is gone

**Comment by kiaking** [comment]
This is fixed with VitePress 0.7.1 👍

**Comment by Topthinking** [comment]
+1 
![image](https://user-images.githubusercontent.com/13382821/177033652-8c15aa73-5bed-4ea5-a73a-4c418ab67730.png)
